recipe/meta.yaml

```
@@ -28,19 +31,19 @@ outputs:
         - setuptools
         - setuptools_scm
       run:
-        - cinnabar ~=0.4.0
+        - cinnabar ~=0.5.0
         - click <8.2
```

```suggestion
        - click
```
pretty sure we fixed this